### PR TITLE
feat(base): creates project base and server

### PR DESCRIPTION
**Pull request by mpiercy827**
What: Setting up project configuration and server for Build Your Own API (BYOAPI).

Why: Can't BYOAPI without project configuration and server.

Details:

- Used generator-lob and npm to create basic project skeleton.
- Added simple HAPI server
- Added nodemon to automatically update project without having to reboot server.

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+5
```
Comment (mgartner) [inline .node-version]: If you wanted to use the latest version of node, you could go with `6.2`

File: config/staging.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```
Comment (mgartner) [inline config/staging.js]: You could add `PORT` since its necessary for starting the server

File: config/production.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```
Comment (mgartner) [inline config/production.js]: You could add `PORT` since its necessary for starting the server

**Comment by mgartner** [comment]
LGTM

**Comment by charlesbjohnson** [comment]
LGTM pending Marcus comments.

**Pull request merged**